### PR TITLE
fix(validations): include commit fees in the change VT in tally trans…

### PR DESCRIPTION
**Pull request by lrubiorod**
…action

### CODE REVIEW

File: validations/src/validations.rs [hunk 1]
```diff
@@ -813,32 +820,45 @@ pub fn validate_tally_outputs<S: ::std::hash::BuildHasher>(
...
-    let reveal_reward = dr_state.data_request.witness_reward;
+    let witness_reward = dr_state.data_request.witness_reward;
+    let reveal_fee = dr_state.data_request.reveal_fee;
+    let commit_fee = dr_state.data_request.commit_fee;
     for (i, output) in ta_tx.outputs.iter().enumerate() {
         if change_required && i == ta_tx.outputs.len() - 1 && output.pkh == dr_state.pkh {
-            let expected_tally_change = reveal_reward * dishonest_len as u64
-                + dr_state.data_request.reveal_fee * (witnesses - n_reveals) as u64;
+            let expected_tally_change = if n_reveals == 0 {
+                witnesses as u64 * (witness_reward + reveal_fee + commit_fee)
```
Comment (tmpolaczyk) [inline validations/src/validations.rs]: Is this correct when `n_reveals == 0` but `n_commits > 0`?

Comment (lrubiorod) [reply validations/src/validations.rs]: O_O my mistake, I modified in other part but forgot here!! Thanks!!

**Comment by tmpolaczyk** [comment]
When #695?

**Comment by lrubiorod** [comment]
> When #695?

The problem with that issue is that we lose information about what happens inside, we only know that the tally created is different from the tally validated, but not why

**Pull request merged**